src/gsi/setuprad.f90

```diff
@@ -1180,6 +1187,18 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
               end do
            end if
 
+           if (abi2km .and. regional) then
+              pred(:,i) = zero
+              if (i>=2 .and. i<=4) then
+                 if (tb_obs(i) > 190.0 .and. tb_obs(i) < 300.0) then
+                    pred(1,i)=1.
+                    pred(2,i)=tb_obs(i)-abi2km_bc(i)
+                    pred(3,i)=(tb_obs(i)-abi2km_bc(i))**2
+                    pred(4,i)=(tb_obs(i)-abi2km_bc(i))**3
```

Comment:
@snebuda I see that you are using pred(2:4,I) for abi2m bias corrections.  
Originally, pred(4:5,i) are predictors for 2nd-order and 1st-order lapse rate correction (air-mass bias correction) respectively.   
You only want 1st-order air-mass bias correction, right?

If you want to keep the original air-mass bias correction, you can consider using pred(7,i) or pred(8,i) for your 3rd-order abi2m correction.  The pred(7:8,i) are reserved for SSMIS.  They are zeros for non-SSMIS sensors.

Reply:
@emilyhcliu Yes, we did not want to include the lapse rate corrections. I can move the term in the 4th slot to 7 or 8 if that is more consistent with other sensors' treatment. Or I can move the 3rd and 4th terms to pred(7:8,i)?

Reply:
@snebuda No, you do not need to move the predictors at all since you do not want the air-mass bias correction.

However, you will need to make sure you turn off the air-mass bias correction terms off completely.  The predoctors pred(4,i) and pred(5,i) are reserved for air-mass bias correction.  You took pred(4,i) for abi2m correction.  You will need to set pred(5,i) to zero. 

```
if (abi2m .and. regional) pred(5,i) = zero
```

Otherwise, you will have a correction from the first-order lapse rate term.
You can checked the abias file from your experiment and see which bias corrections are activated.

Reply:
@emilyhcliu I have pred zeroed out on line 1191 within the "if (abi2km .and. regional) then" block:
pred(:,i) = zero
This new code block is after the lines 1153 & 1154:
           pred(4,i)=tlap*tlap
           pred(5,i)=tlap

Reply:
> @emilyhcliu I have pred zeroed out on line 1191 within the "if (abi2km .and. regional) then" block: pred(:,i) = zero This new code block is after the lines 1153 & 1154: pred(4,i)=tlap*tlap pred(5,i)=tlap

I see.  It is right at the beginning at the section you added.  Got it.  Thanks.